### PR TITLE
Support recent multiarch build on DockerHub

### PR DESCRIPTION
**Pull request by cosmo0920**
Also Dockerfiles are buildable by our laptop with Docker Desktop.

For laptop building:

### For arm64 image

```
$ docker run --rm --privileged multiarch/qemu-user-static:register --reset
$ make DOCKERFILE=v1.8/arm64/debian VERSION=v1.8.0-debian-arm64-1.0
```

### For armhf image

```
$ docker run --rm --privileged multiarch/qemu-user-static:register --reset
$ make DOCKERFILE=v1.8/armhf/debian VERSION=v1.8.0-debian-armhf-1.0
```

### On DockerHub

My testing are succeeded:

**NOTE**: My testing uses _testing_ branch for DockerHub building.

* https://hub.docker.com/repository/docker/cosmo0920/fluentd-docker-image/builds
* https://hub.docker.com/repository/docker/cosmo0920/fluentd-docker-image/tags?page=1

ref: https://github.com/docker/hub-feedback/issues/1261

### CODE REVIEW

File: Dockerfile.template.erb [hunk 1]
```diff
@@ -19,17 +19,31 @@ FROM alpine:3.9
 <% end %>
 <% else %>
 <% if is_armhf %>
+# To set multiarch build for Docker hub automated build.
+FROM golang:alpine AS builder
+WORKDIR /go
+RUN apk add curl --no-cache
+RUN curl -L https://github.com/balena-io/qemu/releases/download/v3.0.0%2Bresin/qemu-3.0.0+resin-arm.tar.gz | tar zxvf - -C . && mv qemu-3.0.0+resin-arm/qemu-arm-static .
```
Comment (ganmacs) [inline Dockerfile.template.erb]: I think It's better to verify sha256sum of this tar file.

Comment (cosmo0920) [reply Dockerfile.template.erb]: Verify sha256sum in https://github.com/fluent/fluentd-docker-image/pull/203/commits/8f8bbb3fe64f9cdffc5feca3282e6955b951b2a2.

**Comment by cosmo0920** [comment]
@repeatedly ?

**Comment by cosmo0920** [comment]
Let's :shipit:!

**Comment by cosmo0920** [comment]
arm64 and armhf images are built successfully! :tada:
https://hub.docker.com/repository/docker/fluent/fluentd/builds

**Pull request merged**